FE/src/components/SearchBar/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { TiDeleteOutline } from "react-icons/ti";
+import {
+  InputSearchBar,
+  SearchBarContainer,
+  SearchInput,
+  SearchButton,
+} from "./style";
+export default function SearchBar({ setInputValue, inputValue }) {
```

Comment:
Um espacinho aqui nao fazia mal a ninguem
```suggestion
import {
  InputSearchBar,
  SearchBarContainer,
  SearchInput,
  SearchButton,
} from "./style";

export default function SearchBar({ setInputValue, inputValue }) {
```